packages/p2p-v2/src/pages/my-profile/screens/MyProfileStats/MyProfileStats.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { useAdvertiserStats } from '../../../../hooks';
+import MyProfileStatsItem from './MyProfileStatsItem';
+import './MyProfileStats.scss';
+
+type MyProfileStatsProps = {
+    advertiserId?: string;
+};
+
+export const MyProfileStats = ({ advertiserId }: MyProfileStatsProps) => {
```

Comment:
just a doubt.. are we following the 'T' prefix naming convention for naming the types like TMyProfileStatsProps.. or just the name of the type?

Reply:
Not sure, in Wallets we don't usually prefix types with `T`